nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -12,5 +12,18 @@ To apply the run-once duration override from the {run-once-operator} to run-once
 
 These release notes track the development of the {run-once-operator} for {product-title}.
 
-:operator-name: The {run-once-operator}
-include::snippets/operator-not-available.adoc[]
\ No newline at end of file
+For an overview of the {run-once-operator}, see xref:../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {run-once-operator}].
+
+[id="run-once-duration-override-operator-release-notes-1-1-0"]
+== {run-once-operator} 1.1.0
```

Comment:
Attribute rendering on the customer portal can be troublesome. I would recommend sticking with a hardcoded name for headings. Can you test the rendering on OV1?